packages/app/src/contacts/components/contacts/contacts.component.test.tsx

```diff
@@ -184,13 +186,15 @@ const renderer = (extraProps?: Partial<Props>) => {
     <VirtuosoMockContext.Provider
       value={{ viewportHeight: 300, itemHeight: 5 }}
     >
-      <Contacts {...props} />
+      <Provider store={store}>
```

Comment:
🟡 why do we need to wrap the Contacts component with redux Provider? I think it's already wrapped in App component